tests/test_activate.py

```diff
@@ -2615,6 +2615,19 @@ def test_legacy_activate_deactivate_cmd_exe(self):
             conda_shlvl = shell.get_env_var('CONDA_SHLVL')
             assert conda_shlvl == '0', conda_shlvl
 
+    @pytest.mark.skipif(not which("cmd.exe"), reason="cmd.exe not installed")
+    def test_tmp_path_with_space_cmd_exe(self):
+        with InteractiveShell("cmd.exe") as shell:
+            tmp = shell.get_env_var("TMP")
+            tmp_with_space = join(tmp, "with space")
+            shell.sendline("SET TMP={}".format(tmp_with_space))
+
+            shell.sendline("activate {}".format(dev_arg))
```

Comment:
```suggestion
            shell.sendline(f"SET TMP={tmp_with_space}")

            shell.sendline(f"activate {dev_arg}")
```
f-strings are great!